src/OpenGraph.php

```diff
@@ -111,7 +111,7 @@ protected function verify_image_url($url)
             ]);
             $headers = get_headers($url, true, $context_headers);
 
-            return stripos($headers[0], '200 OK') ? true : false;
+            return stripos($headers[0], '200') ? true : false;
```

Comment:
@roerlemans  can we keep checks for both '200 OK' and '200'

Reply:
the `OK`-part is not always available in the response. When I check both, it would still fail in the use case I found.